ncolony/process_events.py

```diff
@@ -63,6 +70,8 @@ def remove(self, name):
         """
         self.monitor.removeProcess(name)
         log.msg("Removed monitored process: ", name)
+        for group in self._processToGroups.pop(name):
```

Comment:
we should build a datastructure for this. there's something similar that kurt added to boltons: https://github.com/mahmoud/boltons/pull/137/files#diff-fdeca5472dee680475a26f52e161373c